<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncoderDialog</class>
 <widget class="QDialog" name="EncoderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Column to Encode</string>
  </property>
  <widget class="QLabel" name="ColumnLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Column:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ColumnOptions">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="EncoderLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>17</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Encoder:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="EncoderOptions">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="EncodeButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
